src/Api/EventApi.php

```diff
@@ -56,7 +78,12 @@ public function getList(Calendar $calendar = null, $filter = '', $orderBy = '')
         $url = 'events';
 
         if (null !== $calendar) {
-            $url = sprintf('calendars/%s/events', $calendar->getId());
+            $url = sprintf(
+                'calendars/%s/calendarview?startDateTime=%s&endDateTime=%s',
+                $calendar->getId(),
+                (new \DateTime('now'))->sub(new \DateInterval('P1Y'))->format('Y-m-d\TH:i:s\Z'),
+                (new \DateTime('now'))->add(new \DateInterval('P1Y'))->format('Y-m-d\TH:i:s\Z')
+            );
```

Comment:
the `getList` shouldn't arbitrarly use the `calendarView` if there is a calendar